File: EduSafe.WebApi/Models/Institutions/InstitutionResultEmailEntry.cs
namespace EduSafe.WebApi.Models.Institutions
{
    public class InstitutionResultEmailEntry
    {
        public string RecipientAddress { get; set; }
        public string RecipientName { get; set; }
        public string ResultsPageHtml { get; set; }
        public InstitutionInputEntry InstitutionInputEntry { get; set; }
        public InstitutionOutputSummary InstitutionOutputSummary { get; set; }
    }
}

File: EduSafe.WebSite/src/app/classes/institutions/institutionResultEmailEntry.ts
import { InstitutionInputEntry } from './institutionInputEntry';
import { InstitutionOutputSummary } from './institutionOutputSummary';

export class InstitutionResultEmailEntry {
  RecipientAddress: string;
  RecipientName: string
  ResultsPageHtml: string;
  InstitutionInputEntry: InstitutionInputEntry;
  InstitutionOutputSummary: InstitutionOutputSummary;
}
